profiles/serializer/normalizer.py

```diff
@@ -5,13 +5,13 @@
 
 
 @singledispatch
-def normalize(value: Any) -> str:
-    return str(value)
+def normalize(value: Any) -> Any:
```

Comment:
No need to turn integers into strings. (This should really be string, numeric, dictionary, list, boolean or none.)

Reply:
Could introduce https://github.com/python/typing/issues/182?

Reply:
I think its clear enough as is, these will only be used via the `json` module calls anyway so the expected values should be clear enough. If MyPy aren't pushing it any further at the moment I think we are fine. Interesting topic though.